### ISSUE TITLE
[FasterTransformer Pytorch ] Detect Bert model in Pytorch and replace it by FasterTransformer OP

### ISSUE THREAD

**Issue by wangxiang2713** [NVIDIA/DeepLearningExamples#701]
* *PyTorch* 

Hi, i want to detect Bert model structure in pytorch model and convert the structure by faster transformer op automatically.
Is there any way to access it? Thankyou very much!

**Comment by byshiue** [comment]
Can you describe your requirements in more detail?
What is “detect BERT model structure” and what does “automatically” mean?

**Comment by wangxiang2713** [comment]
> Can you describe your requirements in more detail?
> What is “detect BERT model structure” and what does “automatically” mean?

I mean that if i have one pytorch model which contains BERT model structure and other model structure, i want a program to analysis the model, find the BERT model structure and replace it with faster transformer op.

**Comment by byshiue** [comment]
Sorry, this request is hard to implement. 
It is hard to parse the model by the operations, so we only use the name of tensor to get the weights, and put in the fastertransformer op. 
In your case, you need to replace the BERT model by fastertransformer op manually.

**Comment by wangxiang2713** [comment]
> Sorry, this request is hard to implement.
> It is hard to parse the model by the operations, so we only use the name of tensor to get the weights, and put in the fastertransformer op.
> In your case, you need to replace the BERT model by fastertransformer op manually.

OK, Thankyou very much!
